web-animations/animation-model/keyframe-effects/effect-value-overlapping-keyframes.html

```diff
@@ -21,7 +25,7 @@
                              { offset: 1, opacity: 1 } ],
                            { duration: 1000,
                              easing: 'cubic-bezier(0.5, -0.5, 0.5, 1.5)' });
-  assert_equals(getComputedStyle(div).opacity, '0.2',
+  assert_opacity_value(getComputedStyle(div).opacity, '0.2',
```

Comment:
Don't we want to pass in a number for the expected value here? Or call `parseFloat` on the expected value?

Reply:
Somehow this works, but it would likely be better to use numbers. I will update this in an additional commit.